Drivvn case study draft (#10474)
PostHog/posthog.com: contents/customers/drivvn.md

---
title: 'How Drivvn drives growth with group analytics and feature flags'
customer: Drivvn
logo: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/octomind_logo_673e0ed777.png
logoDark: >-

>   https://res.cloudinary.com/dmukukwp6/image/upload/octomind_logo_dark_a89deeee90.png

Reviewer: Incorrect logos

featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_octomind_bb047603a6.jpg
featuredCustomer: false
industries:

>   - 'Automotive, Agency,SaaS'

Reviewer: ```suggestion
  - 'Automotive, Agency, SaaS'
```

users:
  - Marketing
toolsUsed:
  - Group Analytics
  - Feature Flags
  - Surveys
  - Insights
  - Web analytics

> date: 2024-11-10

Reviewer: This date is incorrect

---

What if shopping for a new vehicle was a user-centered, seamless, thoughtful, online experience — no dealership required? 

> That’s the future Drivvn has been building. Supporting a network of more than 60+ automotive ecommerce providers, Drivvn is the digital engine behind global vehicle brands from Ford to Fiat. Known for creating brand experiences that are design-driven, human-first and still do great at the “selling” part — you’d almost think it was magic.

Reviewer: ```suggestion
That’s the future [Drivvn](https://www.drivvn.com/) has been building. Supporting a network of more than 60+ automotive ecommerce providers, Drivvn is the digital engine behind global vehicle brands from Ford to Fiat. Known for creating brand experiences that are design-driven, human-first and still do great at the “selling” part — you’d almost think it was magic.
```

> Under the hood, Drivvn’s Head of Product Engineering says it’s all PostHog. 

Reviewer: ```suggestion
Under the hood, John Laramy, Drivvn’s Head of Product Engineering, says it’s all PostHog. 
```

John and his team have used PostHog to up their game for over two years. While his team designs and builds online shopping experiences for car brands, PostHog is what helps them optimize and customize the experiences for end users.

> To help you visualize it, as a user, let’s imagine you’re shopping for a Ford vehicle over on this site. Here, you’ll have an easy time comparing vehicles, and shopping for what you need — but what you may not know is that your experience is quickly being customized to better serve you, as you go along.

Reviewer: Are we going to add in the box-out or highlight I suggested here? Up to you, but I think it could add a lot to help visualize Drivvn's concept.

But it helps them go wider than that: as an agency, John and his team hold an ecosystem view. It’s not just one website, one audience base. “It’s also across different brands as well — you might release a feature for one brand, but not enable that across the other ones, until you know how well it's going to perform."

It’s here where PostHog’s group analytics feature comes into play yet again. Since groups will help you analyze trends at an entity-level (such as “companies”), as opposed to a user-level, Drivven is empowered to break data apart by brand, by customer type, and any other segment that becomes relevant for them, for better reporting, and ultimately, better outcomes. 

With this birds-eye view, and the ability to run experiments and changes, John’s team can report to their own customers on how well their branded digital experiences are performing. Through dashboards, the reporting area helps them have those conversations — and it’s the data that is changing people’s minds.

> “With one customer, I would say they were very against this [experimental, data-driven, A/B approach,” says John. “We then did one small-ish change — as subtle as changing the CTA on one page. When we showed the benefit of that, suddenly it was like a lightbulb moment. It clicked: ‘I understand the benefit of that now.’  After that, they were very keen to continue doing that approach.”

Reviewer: ```suggestion
“With one customer, I would say they were very against this experimental, data-driven, A/B approach,” says John. “We then did one small-ish change — as subtle as changing the CTA on one page. When we showed the benefit of that, suddenly it was like a lightbulb moment. It clicked: ‘I understand the benefit of that now.’  After that, they were very keen to continue doing that approach.”
```

<BorderWrapper>
<Quote

>     imageSource="/images/customers/johny_laramy.jpg"

Reviewer: This image hasn't been added yet.

    size="md"
    name="John Laramy"
    title="Head of Product Engineering, Drivvn"
    quote={`“We’ve tried FullStory, MixPanel, Google Analytics and Plausible - but after building out prototypes, trying each tool, PostHog is where I want to be.”`}
/>
</BorderWrapper>

> “One of the reasons we went with PostHog was because of the wide range of features,” says John. “We're using analytics heavily today across all those brand experiences. We're doing lots of feature flags, and also experiments…all towards helping people get to a clear outcome.”

Reviewer: ```suggestion
“One of the reasons we went with PostHog was because of the wide range of features,” says John. “We're using analytics heavily today across all those brand experiences. We're doing lots of feature flags, and also experiments - all towards helping people get to a clear outcome.”
```

> As someone who has been in engineering roles for more than a decade, John cares about the details. “We’ve used other tools like [MixPanel](https://posthog.com/blog/posthog-vs-mixpanel) and [Google Analytics](https://posthog.com/blog/posthog-vs-ga4), but I think the real thing that we were after was that it was third-party and cookieless.”

Reviewer: ```suggestion
As someone who has been in engineering roles for more than a decade, John cares about the details. “We’ve used other tools like [Mixpanel](/blog/posthog-vs-mixpanel) and [Google Analytics](/blog/posthog-vs-ga4), but I think the real thing that we were after was that it was third-party and [cookieless](/tutorials/cookieless-tracking).”
```

Internal links should be relative URLs

“We're big advocates of PostHog at Drivvn — it's not just myself,” John continues. “There's a number of other people who are in PostHog on a daily basis looking at that good stuff. The [documentation](/docs) is very good. You guys are obviously releasing new features often. When we do have questions, we use Slack and we're able to reach out to you, and we get a response in a timely fashion.”

What’s next for John and Drivvn? Maxing out the system. “I think we are utilizing maybe 70% of the new features,” he says. “We've only very recently started to play with the data pipelines —it’s something that we'd like to play with in the future, as having these rich insights is really helpful.”

> <BorderWrapper>
> <Quote
>     imageSource="/images/customers/johny_laramy.jpg"
>     size="md"
>     name="John Laramy"
>     title="Head of Product Engineering, Drivvn"
>     quote={`“We’ve tried FullStory, MixPanel, Google Analytics and Plausible - but after building out prototypes, trying each tool, PostHog is where I want to be.”`}
> />
> </BorderWrapper>

Reviewer: ```suggestion
```
This is a duplicate
